src/WarriorRogueMage/CharacterCreation/SetAttributesState.cs

```diff
@@ -157,22 +157,22 @@ private void SetAttributeTarget(ref int targetPoints, int newValue)
         private void RefreshScreen(bool sendPrompt = true)
         {
             var sb = new StringBuilder();
-            sb.AppendLine();
-            sb.AppendLine();
-            sb.AppendLine("You have 10 character points to be divided between 3 attributes.");
-            sb.AppendLine("No attribute can have more than 6 points. Attributes can be zero.");
-            sb.AppendLine();
-            sb.AppendLine(string.Format("Warrior : {0}", warriorPoints));
-            sb.AppendLine(string.Format("Rogue   : {0}", roguePoints));
-            sb.AppendLine(string.Format("Mage    : {0}", magePoints));
-            sb.AppendLine();
-            sb.AppendFormat("You have {0} character points left.", MaxPoints - SpentPoints);
-            sb.AppendLine();
-            sb.AppendLine("<%yellow%>====================================================================");
-            sb.AppendLine("To add points to an attribute, use the + operator. Example: warrior +6");
-            sb.AppendLine("To subtract points from an attribute, use the - operator. Example: warrior -6");
-            sb.AppendLine("When you are done distributing the character points, type done.");
-            sb.AppendLine("====================================================================<%n%>");
+            sb.AppendAnsiLine();
+            sb.AppendAnsiLine();
+            sb.AppendAnsiLine("You have 10 character points to be divided between 3 attributes.");
+            sb.AppendAnsiLine("No attribute can have more than 6 points. Attributes can be zero.");
+            sb.AppendAnsiLine();
+            sb.AppendAnsiLine($"Warrior : {warriorPoints}" + AnsiSequences.NewLine);
```

Comment:
nit: Could further improve these with the embedded newline `{warriorPoints}{AnsiSequences.NewLine}");` if you want.